apps/journeys/pages/[journeyId].tsx

```diff
@@ -20,6 +20,7 @@ import {
   STEP_FIELDS,
   RADIO_OPTION_FIELDS,
   RADIO_QUESTION_FIELDS,
+  TRIGGER_FIELDS,
```

Comment:
```suggestion
  VIDEO_TRIGGER_FIELDS,
```

same changes on backend should be made here (rename trigger to video trigger).